### ISSUE TITLE
Should update feed URL for 301 Moved Permanently

### ISSUE THREAD

**Issue by danielpunkass** [brentsimmons/NetNewsWire#953]
Some of my older feeds are just showing up as empty with no error in NNW5. I think that 301 Moved Permanently should be treated as instruction to update the feed URL in NNW's feed list. Here's an example of a 301 Moved Permanently feed URL:

http://www.yourarlington.com/?format=feed&type=rss

**Comment by brentsimmons** [comment]
You’re right about this. But it’s a tricky situation, because it also means figuring out what to do with articles, whose IDs are matched to the original feed URL.
